source/tyr/api_v1.md

```diff
@@ -56,7 +57,9 @@ If lots of elements expected in the response:
 }
 ```
 
-- PUT /v1/<endpoints>:
+- PUT /v1/<endpoints>:  
+Used to update a new element of the endpoint. Returned status code should **200**.  
+Note: If the POST method isn't available for an endpoint, PUT can be used to create a new element. In this case, the returned status code should be **201**
```

Comment:
:cold_sweat:
:+1: Nice of you to document it, though!

Reply:
Should we create a method POST for every endpoint? It can be discussed, in order to have an even better homogeneity for this v1.
However, we can note this (create a ticket?) for a future refinement and not consider it blocking for this PR.

Reply:
I created a ticket to track what is left for the full v1 experience:
https://jira.kisio.org/browse/NAVP-1425